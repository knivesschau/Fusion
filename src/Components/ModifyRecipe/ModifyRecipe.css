.Modify_Recipe {
    text-align: center; 
    margin: 1em;
}

#submit-mod, #restart-mod {
    margin: .5em;
    padding: .25em;
    margin-bottom: 2em;
}