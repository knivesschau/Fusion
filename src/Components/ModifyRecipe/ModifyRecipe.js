import React, { Component } from 'react';
import fusionContext from '../../fusionContext';
import './ModifyRecipe.css';

export default class ModifyRecipe extends Component {
    render() {
        return (
            <></>
        );
    };
}