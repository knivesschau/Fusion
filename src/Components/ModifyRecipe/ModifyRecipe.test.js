import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import ModifyRecipe from './ModifyRecipe';

describe ('ModifyRecipe Component', () => {
    it ('Renders the complete ModifyRecipe Component and all nested Components', () => {
        const wrapper = shallow(<ModifyRecipe/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});