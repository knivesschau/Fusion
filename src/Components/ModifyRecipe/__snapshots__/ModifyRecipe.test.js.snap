// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModifyRecipe Component Renders the complete ModifyRecipe Component and all nested Components 1`] = `<Fragment />`;
